superset/viz.py

```diff
@@ -484,7 +497,7 @@ def get_payload(self, query_obj: Optional[QueryObjectDict] = None) -> VizPayload
         payload["rejected_filters"] = [
             {"reason": "not_in_datasource", "column": col}
             for col in filter_columns
-            if col not in columns
+            if col not in columns and col not in filter_values_columns
```

Comment:
Should they be added to `applied_filters` if removed from `rejected_filters`?

Reply:
fixed.